usaspending_api/references/v2/views/glossary.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.response import Response
+from rest_framework.request import Request
+
+from usaspending_api.common.helpers.generic_helper import get_pagination
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.views import APIDocumentationView
+from usaspending_api.common.serializers import LimitableSerializer
+from usaspending_api.references.models import Definition
+
+
+class DefinitionSerializer(LimitableSerializer):
+
+    class Meta:
+
+        model = Definition
+        fields = ['term', 'slug', 'data_act_term', 'plain', 'official', 'resources']
+
+
+class GlossaryViewSet(APIDocumentationView):
+    """
+    This view returns paginated glossary terms in a list
+    """
+
+    @cache_response()
+    def get(self, request: Request) -> Response:
+        """
+        Accepts only pagination-related query parameters
+        """
+
+        get_request = request.query_params
+        limit = get_request.get('limit', 500)
```

Comment:
Surprised you didn't use Tinyshield. HTTP 500 when a non-integer value is passed as the limit or page